documentation/src/main/asciidoc/topics/con_off_heap_memory.adoc

```diff
@@ -0,0 +1,66 @@
+[id='off_heap_memory']
+= Storing Data in Off Heap Memory
+{brandname} can store cache entries in JVM heap memory or off heap.
+Off heap storage lets you use unmanaged, native memory space for the JVM.
```

Comment:
```suggestion
Off heap storage lets you use unmanaged, native memory space for the JVM.
```
Off heap storage lets your objects occupy native memory outside of the managed JVM memory.